content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -296,30 +292,32 @@ Once you have rolled out the container name change to the workload resource, tid
 the old container name from the HPA specification.
 {{< /note >}}
 
-## Support for multiple metrics
 
-Kubernetes 1.6 adds support for scaling based on multiple metrics. You can use the `autoscaling/v2` API
-version to specify multiple metrics for the Horizontal Pod Autoscaler to scale on. Then, the Horizontal Pod
-Autoscaler controller will evaluate each metric, and propose a new scale based on that metric. The largest of the
-proposed scales will be used as the new scale.
+## Scaling on custom metrics
 
-## Support for custom metrics
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
 
-{{< note >}}
-Kubernetes 1.2 added alpha support for scaling based on application-specific metrics using special annotations.
-Support for these annotations was removed in Kubernetes 1.6 in favor of the new autoscaling API.  While the old method for collecting
-custom metrics is still available, these metrics will not be available for use by the Horizontal Pod Autoscaler, and the former
-annotations for specifying which custom metrics to scale on are no longer honored by the Horizontal Pod Autoscaler controller.
-{{< /note >}}
+(the `autoscaling/v2beta2` API version previously provided this ability as a beta feature)
 
-You can also use a HorizontalPodAutoscaler to change the scale of a
-workload based on custom metrics.  You can add custom metrics for the
-Horizontal Pod Autoscaler to use in the `autoscaling/v2` API.
-Kubernetes then queries the new custom metrics API to fetch the values
-of the appropriate custom metrics.
+Provided that you use the `autoscaling/v2` API version, you can configure a HorizontalPodAutoscaler
+to scale based on a custom metric (that is not built in to Kubernetes or any Kubernetes component).
```

Comment:
```suggestion
to scale based on a custom metric (that is not built into Kubernetes or any Kubernetes component).
```

Reply:
Not sure about this one. The phrase here is “built in”.

Reply:
That's just how it struck me. Fine to leave it.